app/assets/v2/css/profile.css

```diff
@@ -1,439 +1,440 @@
-  filter: invert(100%);
-}
-
-.float_avatar_image {
-  margin-top: -140px;
-  position: relative;
-}
-
-.float_avatar_image img {
-  height: 186px;
-  width: 186px;
-  border: 3px solid #FFFFFF;
-  box-shadow: 0 -2px 4px 0 rgba(0,0,0,0.50);
-}
-
-.avatar_edit {
-  text-align: center;
-  background: linear-gradient(to top,rgba(255,255,255,0.6),rgba(255,255,255,0.0));
-  border-radius: 0% 0% 50% 50% / 0% 0% 100% 100%;
-  color: black;
-  height: 90px;
-  width: 180px;
-  position: absolute;
-  top: 93px;
-  margin: auto;
-  right: 0;
-  left: 0;
-  transition: .2s all linear;
-}
-
-.avatar_edit_text {
-  position: absolute;
-  top: 50%;
-  color: black;
-  font-weight: bold;
-  font-size: 20px;
-  margin: auto;
-  width: 100%;
-  height: 40px;
-}
-
-.is_my_profile a:hover .profile-header__avatar{
-  border: 3px solid #0fce7c;
-}
-
-.is_my_profile{
-  cursor: pointer;
-}
-
-a:hover .avatar_edit {
-  background-color: rgba(255, 255, 255, 0.7);
-  height: 180px;
-  width: 180px;
-  top: 3px;
-  border-radius: 50%;
-}
-
-a:hover .avatar_edit .avatar_edit_text {
-  top: 40%;
-}
-
-.picky img {
-  width: 100%;
-  height: auto;
-  vertical-align: middle;
-}
-
-
-.profile-organizations .organization {
-  display: inline-block;
-}
-
-.profile-bounties--activities{
-  min-height: 200px;
-}
-
-.profile-organizations .organization img{
-  max-height: 4rem;
-  max-width: 4rem;
-  margin-right: 0.5rem;
-  margin-bottom: 0.5rem;
-  border-radius: 50%;
-}
-
-
-@media (max-width: 1200px) {
-  .profile-header{
-    min-height: 350px;
-  }
-}
-
-@media (max-width: 767.98px) {
-
-  .profile-header__main-infos {
-    text-align: center;
-  }
-
-  .profile-header{
-    min-height: 200px;
-  }
-
+	min-height: 150px;
+  }
+  
+  .overflow-visible {
+	overflow: visible;
+  }
+  
+  .profile-card {
+	background: white;
+	border-radius: 5px;
+	z-index: 1;
+	position: relative;
+	padding: 2rem;
+  }
+  
   .profile-header__avatar {
-    float: none;
-    margin-right: 0!important;
+	float: left;
+	width: 7.25rem;
+	height: 7.25rem;
+	border-radius: 50%;
   }
-
+  
+  .profile-header {
+	position: relative;
+	min-height: 450px;
+  }
+  
   .profile-header__handle {
-    display: block;
-  }
-
-  .profile-header__links li {
-    text-align: center;
-    float: none;
-    margin-right: 0;
-  }
-
-  .row.bottom_notification{
-    display: none;
-  }
-
-}
-
-
-.profile-header__stats {
-  border: none;
-}
-
-.profile-header__stats .card-header {
-  padding: 0.1rem 0.7rem;
-  color: #666;
-  border-radius: 0;
-  border-bottom: none;
-  font-size: 1.2rem;
-}
-
-.profile-header__stats .card-header b {
-  color: #000;
-  font-size: 1.7rem;
-}
-
-.profile-header__stats .card-body {
-  padding: 0.7rem;
-}
-
-@media (min-width: 992px) and (max-width: 1199.98px) {
-
+	padding-top: 0.8rem;
+	margin-bottom: 0;
+	line-height: 1;
+	font-weight: 700;
+	font-size: 1.7rem;
+  }
+  
+  .profile-header__handle small {
+	text-transform: none;
+  }
+  
+  .profile-header__bio {
```

Comment:
are these all just linting changes? did anything change in this css file?  any way to make them *not* show up in the diff?

Reply:
I can't reproduce this locally

Reply:
what command are you using to try to repo this locally?  if you diff your branch and the master branch it should show up locally